tests/static/test_mongo_config.json

```diff
@@ -1,5 +1,6 @@
 {
     "debug": true,
+    "query_group": null,
```

Comment:
By adding it explicitly to the `test_config.json` file (as `null`) and not adding it explicitly here, you're also testing that it works for these two cases (i.e., you'll be adding the test of "what if the option is not supplied - will it still correctly instantiate as `null`?").
```suggestion
```